src/lib/components/View.react.js

```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { View as VtkItem } from 'react-vtk-js';
+import VtkItem from 'react-vtk-js/dist/esm/core/View';
```

Comment:
A better import should be `import VtkItem from 'react-vtk-js/core/View.js';`

Reply:
I don't think that works

Reply:
```
Critical dependency: the request of a dependency is an expression
 @ ./node_modules/react-vtk-js/dist/esm/node_modules/vtk.js/dist/esm/vendor/readable-stream/lib/_stream_duplex.js
 @ ./node_modules/react-vtk-js/dist/esm/node_modules/vtk.js/dist/esm/Filters/Sources/ConcentricCylinderSource.js
 @ ./node_modules/react-vtk-js/dist/esm/index.js
 @ ./src/lib/components/Algorithm.react.js
 @ ./src/lib/index.js

ERROR in ./src/lib/components/View.react.js
Module not found: Error: Can't resolve 'react-vtk-js/core/View' in '/home/xhlu/dev/dash-vtk/src/lib/components'
 @ ./src/lib/components/View.react.js 4:0-45 15:42-49
 @ ./src/lib/index.js
npm ERR! code ELIFECYCLE
npm ERR! errno 2
```

Reply:
Import the usual `react-vtk-js` instead. The way these esm modules are published isn't compatible with our usage. You can see above that the code is looking for `./node_modules/react-vtk-js/dist/esm/node_modules/vtk.js` but that folder does not exist in the released package, see: https://unpkg.com/browse/react-vtk-js@1.2.1/dist/esm/node_modules/

Reply:
@jourdain ^^

Reply:
I'll see what I can do to move the required devDep of vtk.js into dep.

Reply:
Just pushed `react-vtk-js@1.2.2` for you to try out. Normally the esm bundle should have all its dependency.